ddtrace/_trace/processor/__init__.py

```diff
@@ -189,6 +196,10 @@ def on_span_finish(self, span):
             span.set_metric("_dd.top_level", 1)
 
 
+def _update_span_tags_from_context(span, context):
+    span._context._update_tags(span)
```

Comment:
```suggestion
    context._update_tags(span)
```
I'm taking a guess - was this the logic you intended here?